packages/repco-frontend/app/routes/playlists.tsx

```diff
@@ -0,0 +1,67 @@
+import type { ActionFunction, LoaderFunction } from '@remix-run/node'
```

Comment:
move some parts of this page to /$playlists/index and just leave the shared parts here and use the outlet-component